CRM/Inventoryfield/Upgrader.php

```diff
@@ -56,17 +56,16 @@ public function install() {
   // }
 
   /**
-   * Example: Run a couple simple queries.
+   * Add column civicrm_inventoryfield.limit_per_amount
    *
    * @return TRUE on success
    * @throws Exception
    */
-  // public function upgrade_4200() {
-  //   $this->ctx->log->info('Applying update 4200');
-  //   CRM_Core_DAO::executeQuery('UPDATE foo SET bar = "whiz"');
-  //   CRM_Core_DAO::executeQuery('DELETE FROM bang WHERE willy = wonka(2)');
-  //   return TRUE;
-  // }
+   public function upgrade_4200() {
```

Comment:
@lucky091588 Note for future: when making changes to the database tables (DAO and/or schema xml) we also need an upgrader function like this so that civicrm will apply those schema changes on existing installations. Right now I can't the official civicrm documentation about such hooks, but it works very much like [hook_update_N in Drupal 7](https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7.x). Please ask me some time for an explanation if you have questions.